package.json

```diff
@@ -29,14 +29,17 @@
   ],
   "dependencies": {
     "got": "~9.6.0",
-    "tldts": "~5.6.3"
+    "tldts": "~5.6.3",
+    "tldts-experimental": "~5.6.3"
```

Comment:
@remusao why use `tldts-experimental`?

I prefer to keep `tldts` as a dependency since it's used on the rest of folks (metascraper, browserless) and it will deduped under npm install

Reply:
Makes sense, totally up to you. The difference is that `tldts-experimental` is faster (although `tldts` is already extremely fast) and the bundle loads faster because it uses a compact typed array instead of a JavaScript object to store the data.